src/en/components/notice/code.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Notice
+layout: 'layouts/component-documentation.njk'
+translationKey: 'noticeCode'
+tags: ['noticeEN', 'code']
+date: "git Last Modified"
+---
+
+## Build a Notice
+
+Use the notice component to deliver important context-specific messages to people. The notice communicates varying levels of urgency and meaning using content and visual cues.
+
+## Accessibility and Code Guidance
+
+### Define the type of notice
+
+Use the `type` attribute to categorize the notice as ‘Info’, ‘Warning’, ‘Danger’ or ‘Success’. This choice determines the visual styling of the notice and communicates the urgency or importance of the message to people.
```

Comment:
Should we flag ‘Info’, ‘Warning’, ‘Danger’ or ‘Success’ as code and make it lowercase to match the values required to pass to the `type` prop?

Reply:
Here's what we've done especially when it involves getting the french translation for some items that are best left as code:

```suggestion
Use the `type` attribute to categorize the notice as `info` (Info, `warning` (Warning), `danger` (Danger) or `Success` (Success). This choice determines the visual styling of the notice and communicates the urgency or importance of the message to people.
```

Reply:
@daine can we also change `Success` to lowercase please to match the code?

Reply:
Looking at it again - can we skip the text in brackets for EN since it's the same word?

Reply:
Round 2 of this suggestion:

```suggestion
Use the `type` attribute to categorize the notice as `info`, `warning`, `danger` or `success`. This choice determines the visual styling of the notice and communicates the urgency or importance of the message to people.
```